### ISSUE TITLE
some errors in the cross-validation step.

### ISSUE THREAD

**Issue by ming616** [glisanti/MCK-CCA#2]
hi,
I'm interested in this project. When I tried the demo, I got some errors in the cross-validation step. 
Would you mind give me some instructions to correct them?
I ran it with matlab 2017b in windows 10 x64 and errors below :
...
....
Learning KCCA [Desc 20: Kernel ExpChi]
Learning KCCA [Desc 20: Kernel Chi2]
Cross-validating LR bias and C...
Starting parallel pool (parpool) using the 'local' profile ...
connected to 6 workers.
Analyzing and transferring files to the workers ...done.
Error using cross_validate_c_bias (line 34)
An UndefinedFunction error was thrown on the workers for 'train'.  This might be because the file containing 'train' is not accessible
on the workers.  Use addAttachedFiles(pool, files) to specify the required files to be attached.  See the documentation for
'parallel.Pool/addAttachedFiles' for more details.

Error in iterativeLR (line 102)
    [lr_model, lr_params, ind_to_rem_tmp] = cross_validate_c_bias(-kcca_featAll,train_labelsAll,lr_params);

Error in Demo_MCKCCA (line 95)
        iterativeLR(train_a,train_b,num_train,idxTrain1,idxTrain2,params,lr_params);

Caused by:
    Undefined function 'train' for input arguments of type 'double'.

**Comment by svebk** [comment]
Hi @ming616, that seems similar to the previous issue reported. Are you using the latest commit? Can you pull and try again if you weren't?

**Comment by ming616** [comment]
OK, got it. many thanks!

**Comment by ming616** [comment]
Hi @svebk , maybe I found the key to this issue. 
I found your latest commit still give same errors in my dev envirement. So i considered that errors should be caused by inconsistent environmental factors.
After I replaced the default gcc and g++ version in linux (I'm using R2017b on Ubuntu 16.04) to used in MATLAB2017B, the gcc-4.9 and g++-4.9, it begin to work. 👍  
 I just followed this direction:  https://stackoverflow.com/questions/40352237/using-previous-version-of-gcc-for-mex-in-matlab
